### DISCUSSION TITLE
zigbee2mqtt Failed to interview

### DISCUSSION THREAD

**Discussion by scontomio** [indomus/forum#2269] [ZigBee, Z-Wave, Thread, Matter]
Ciao, ho seguito le vostre fantastiche guide e sono riuscito a far funzionare sul mio tablet (linux) con home assistant la chiavetta ZBDongle-E con Zigbee2MQTT.

Tramite Zigbee2MQTT sono riuscito a collegare senza problemi https://www.zigbee2mqtt.io/devices/E2007.html, adesso ho provato 3 ore il collegamento di https://www.zigbee2mqtt.io/devices/AQSZB-110.html.

Ricevevo sempre questo errore:
Failed to interview '0x0015bc0036000fa4', device has not successfully been paired

Ho provato a mettere una prolunga usb, cambiare canale radio, resettare il device e ricollegarlo 20 volte ma ricevo sempre questo errore... Idee di cosa poter provar a fare? Non penso sia il device rotto...

Grazie!

**Comment by indomus** [comment]
> Ciao, ho seguito le vostre fantastiche guide e sono riuscito a far funzionare sul mio tablet (linux) con home assistant la chiavetta ZBDongle-E con Zigbee2MQTT.
> 
> Tramite Zigbee2MQTT sono riuscito a collegare senza problemi https://www.zigbee2mqtt.io/devices/E2007.html, adesso ho provato 3 ore il collegamento di https://www.zigbee2mqtt.io/devices/AQSZB-110.html.
> 
> Ricevevo sempre questo errore: Failed to interview '0x0015bc0036000fa4', device has not successfully been paired
> 
> Ho provato a mettere una prolunga usb, cambiare canale radio, resettare il device e ricollegarlo 20 volte ma ricevo sempre questo errore... Idee di cosa poter provar a fare? Non penso sia il device rotto...
> 
> Grazie!

Strano perché quel device lo abbiamo [recensito con successo](https://indomus.it/componenti/sensori/develco-frient-air-quality-sensor-sensore-ambientale-e-di-qualita-dellaria/) proprio con ZigBee2MQTT (e altri).
Prova a cambiare le batterie, [effettuare il pairing ](https://www.elektroimportoren.no/docs/lib/4514806-Manual-18.pdf) lasciandolo a pochi centimetri dall'antenna coordinator, verifica prima di avere l'ultimo firmware di quest'ultima. Altri test non ce ne vengono in mente.

**Comment by scontomio** [comment]
Infatti avevo visto la vostra recensione e mi aveva convinto! Cambiare pile fatto senza successo, pairing da molto vicino anche.

Per quanto riguarda ZBDongle-E ho seguito la vostra guida (fantastica) ed installato l'ultimo firmware.. Potrebbe essere che ZBDongle-E non è supportato ufficialmente da Zigbee2MQTT e ci sono problemi? 

Il dispositivo guasto mi sembra difficile visto che viene comunque rilevato...

<img width="808" alt="Screenshot 2023-01-11 alle 10 13 31" src="https://user-images.githubusercontent.com/96835210/211765664-4ffe75fb-8719-4851-aa3c-7c29b12bdb7d.png">

**Reply by snakuzzo** [reply]
potrebbe essere tutto...
Potresti mettere in debug zigbee2mqtt come descritto [QUI](https://www.zigbee2mqtt.io/guide/usage/debug.html), ma prima io proverei cambiando canale (però perderesti il pairing degli altri dispositivi)

**Reply by indomus** [reply]
> Infatti avevo visto la vostra recensione e mi aveva convinto! Cambiare pile fatto senza successo, pairing da molto vicino anche.
> 
> Per quanto riguarda ZBDongle-E ho seguito la vostra guida (fantastica) ed installato l'ultimo firmware.. Potrebbe essere che ZBDongle-E non è supportato ufficialmente da Zigbee2MQTT e ci sono problemi?
> 
> Il dispositivo guasto mi sembra difficile visto che viene comunque rilevato...
> 
> <img alt="Screenshot 2023-01-11 alle 10 13 31" width="808" src="https://user-images.githubusercontent.com/96835210/211765664-4ffe75fb-8719-4851-aa3c-7c29b12bdb7d.png">

mah, difficile dirlo, ma saresti il primo con la -E ad avere problemi di questo tipo su z2m...

**Comment by scontomio** [comment]
ho provato a cambiare canale, si è disconnesso l'altro device zigbee e poi rifacendo il pairing è andato, mentre il Develco AQSZB-110 da sempre lo stesso errore.. Proverò anche la modalità debug, ho scritto anche al produttore, ma se di oggi non ne esco faccio il reso, è un prodotto comunque parecchio costoso e mi pare strano non si integri subito...

**Reply by indomus** [reply]
(ci sta che sia guasto, eh)

**Reply by scontomio** [reply]
richiesta sostituzione ad Amazon, domani arriva! vi faccio sapere :) Grazie!

**Comment by scontomio** [comment]
secondo device arrivato e... stesso problema dell'altro, quindi non penso proprio sia rotto.. 

Ho provato a riflashare anche il firmware della chiavetta ma non è cambiato nulla.. 

Come posso trasformare questo docker run per avere il debug? Dicono di mettere un `-e DEBUG=zigbee-herdsman*`, ma se lo aggiungo si spacca

`docker run --init -d --name="zigbee2mqtt" -e TZ=Europe/Rome -v /home/deb/zigbee2mqtt:/app/data --device=/dev/ttyACM0 --network host --restart always koenkk/zigbee2mqtt`

**Reply by snakuzzo** [reply]
Non si spacca

```
docker run --init -d --name="zigbee2mqtt" -e TZ=Europe/Rome -e DEBUG=zigbee-herdsman* -v /home/deb/zigbee2mqtt:/app/data --device=/dev/ttyACM0 --network host --restart always koenkk/zigbee2mqtt
```

**Reply by scontomio** [reply]
è vero così funziona! adesso ho il log completo, ho aperto un "bug" su zigbee2mqtt
[log.txt](https://github.com/indomus/forum/files/10411328/log.txt)

**Reply by scontomio** [reply]
oggi ho aggiunto altri due sensori zigbee e si sono collegati senza problemi. A me pare un problema di incompatibilità, ho provato ad aprire un ticket direttamente su zigbee2mqtt :) Speriamo, non ho trovato altri sensori di qualità dell'aria se non al doppio del prezzo
